src/types/contact-request.ts

```diff
@@ -1,10 +1,10 @@
 /*
-    NOTE: 
+    NOTE:
         I am choosing to use an interface here for simplicity.
         You may choose to create a Type instead and that's okay!
         https://www.typescriptlang.org/play/?#example/types-vs-interfaces
 */
-export interface IContactRequest {
+export interface ContactRequest {
```

Comment:
Nit, but for peak colloquiality, pls don't prefix with `I` or `T` 💀

Reply:
I figured you'd say that haha. I'm not passionate about it one way or another. 😄